src/pages/List/List.js

```diff
@@ -1,7 +1,75 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import CardList from './Card/CardList';
+import Filter from './Filter/Filter';
+import styled from 'styled-components';
 
 const List = () => {
-  return <div>List</div>;
+  const [personCount, setPersonCount] = useState(0);
+  const [cardList, setCardList] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/listFilter.json')
+      .then(res => res.json())
+      .then(data => {
+        setCardList(data);
+      });
+  }, []);
+
+  const increaseCount = () => {
+    setPersonCount(() => personCount + 1);
+  };
+  const decreaseCount = () => {
+    setPersonCount(() => personCount - 1);
+    if (personCount === 0) {
+      setPersonCount(0);
+    }
```

Comment:
```suggestion
  const decreaseCount = () => {
    if (personCount === 0) return;
    setPersonCount(() => personCount - 1);
```
* 만약 count가 0 이라면 먼저 return으로 막아버릴 수 있습니다.
* https://jheloper.github.io/2019/06/write-early-return-code/ 이 링크를 참고 해 주세요 :-)

Reply:
제 민망한 코드에 한줄기 같은 빛...